src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -6,55 +6,80 @@
 import core.basesyntax.figures.RightTriangle;
 import core.basesyntax.figures.Square;
 import core.basesyntax.properties.Figure;
+import core.basesyntax.state.Color;
 import java.util.Random;
 
 public class FigureSupplier {
     private static final int FIGURE_COUNT = 4;
     private static final int MEASUREMENT = 10;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final Color DEFAULT_CIRCLE_COLOR = Color.WHITE;
     private final Random random = new Random();
     private final ColorSupplier colorSupplier = new ColorSupplier();
 
     public Figure getRandomFigure() {
         int figureNumber = random.nextInt(FIGURE_COUNT);
         switch (figureNumber) {
             case 0:
-                Circle circle = new Circle();
-                circle.setColor(colorSupplier.getRandomColor());
-                circle.setRadius(random.nextInt(MEASUREMENT) + 1);
-                return circle;
+                return createRandomCircle();
 
             case 1:
-                IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
-                isoscelesTrapezoid.setColor(colorSupplier.getRandomColor());
-                isoscelesTrapezoid.setHeight(random.nextInt(MEASUREMENT) + 1);
-                isoscelesTrapezoid.setBottomBase(random.nextInt(MEASUREMENT) + 1);
-                isoscelesTrapezoid.setTopBase(random.nextInt(MEASUREMENT) + 1);
-                return isoscelesTrapezoid;
+                return createRandomisoscelesTrapezoid();
 
             case 2:
-                Rectangle rectangle = new Rectangle();
-                rectangle.setColor(colorSupplier.getRandomColor());
-                rectangle.setFirstSide(random.nextInt(MEASUREMENT) + 1);
-                rectangle.setSecondSide(random.nextInt(MEASUREMENT) + 1);
-                return rectangle;
+                return createRandomRectangle();
 
             case 3:
-                RightTriangle rightTriangle = new RightTriangle();
-                rightTriangle.setColor(colorSupplier.getRandomColor());
-                rightTriangle.setFirstLeg(random.nextInt(MEASUREMENT) + 1);
-                rightTriangle.setSecondLeg(random.nextInt(MEASUREMENT) + 1);
-                return rightTriangle;
+                return createRandomRightTriangle();
 
             default:
-                Square square = new Square();
-                square.setColor(colorSupplier.getRandomColor());
-                square.setSide(random.nextInt(MEASUREMENT) + 1);
-                return square;
-
+                return createRandomSquare();

```

Comment:
default case have to create default figure. 
Also add one more case to create random square